kie-bom/pom.xml

```diff
@@ -385,6 +385,19 @@
         <classifier>sources</classifier>
       </dependency>
 
+      <!-- KIE Workbench Playground Repository -->
+      <dependency>
+        <groupId>org.kie.workbench.playground</groupId>
+        <artifactId>playground-parent</artifactId>
+        <version>${version.org.kie}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.kie.workbench.playground</groupId>
+        <artifactId>playground-parent</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>sources</classifier>
```

Comment:
I believe this is not needed, because the `playground-parent` uses packaging `pom` which does not produce binary jar nor sources jar.